w_flux_codemod/test/action_v2_suggestor_test.dart

```diff
@@ -0,0 +1,246 @@
+          context,
+          expectedOutput,
+        );
+      });
+    }
+
+    group('ActionV2DispatchMigrator', () {
+      Suggestor suggestor() => ActionV2DispatchMigrator();
+      testSuggestor(
+        'Function Expression Invocation',
+        suggestor,
+        'class C { ActionV2 action; } void main() { C().action(); }',
+        'class C { ActionV2 action; } void main() { C().action(null); }',
+      );
+      testSuggestor(
+        'Function Expression Invocation type 2',
+        suggestor,
+        'void main() { var a = ActionV2(); a(); }',
+        'void main() { var a = ActionV2(); a(null); }',
+      );
+    });
+
+    group('FieldAndVariableMigrator', () {
+      Suggestor suggestor() => ActionV2FieldAndVariableMigrator();
+      group('VariableDeclarationList', () {
+        testSuggestor(
+          'with type, no intializer',
+          suggestor,
+          'Action<num> action;',
+          'ActionV2<num> action;',
+        );
+        testSuggestor(
+          'with type and intializer',
+          suggestor,
+          'Action<num> action = Action<num>();',
+          'ActionV2<num> action = ActionV2<num>();',
+        );
+        testSuggestor(
+          'no type, with intializer',
+          suggestor,
+          'var action = Action<num>();',
+          'var action = ActionV2<num>();',
+        );
+        testSuggestor(
+          'dynamic Actions',
+          suggestor,
+          'Action a; Action b = Action(); var c = Action();',
+          'ActionV2 a; ActionV2 b = ActionV2(); var c = ActionV2();',
+        );
+      });
+      group('FieldDeclaration', () {
+        testSuggestor(
+          'with type, no intializer',
+          suggestor,
+          'class C { Action<num> action; }',
+          'class C { ActionV2<num> action; }',
+        );
+        testSuggestor(
+          'with type and intializer',
+          suggestor,
+          'class C { Action<num> action = Action<num>(); }',
+          'class C { ActionV2<num> action = ActionV2<num>(); }',
+        );
+        testSuggestor(
+          'no type, with intializer',
+          suggestor,
+          'class C { var action = Action<num>(); }',
+          'class C { var action = ActionV2<num>(); }',
+        );
+        testSuggestor(
+          'dynamic Actions',
+          suggestor,
+          'class C { Action a; Action b = Action(); var c = Action(); }',
+          'class C { ActionV2 a; ActionV2 b = ActionV2(); var c = ActionV2(); }',
+        );
+      });
+      group('InstanceCreationExpression', () {
+        testSuggestor(
+          'variable initialization',
+          suggestor,
+          'var action; action = Action();',
+          'var action; action = ActionV2();',
+        );
+        testSuggestor(
+          'field initialization',
+          suggestor,
+          'class C { var action; C() { action = Action(); } }',
+          'class C { var action; C() { action = ActionV2(); } }',
+        );
+        testSuggestor(
```

Comment:
@evanweible-wf what was the original intention with this test? Should the suggestor actually be skipping the dynamic functions?